src/vendor/core/warning.js

```diff
@@ -31,6 +31,10 @@ if (__DEV__) {
       );
     }
 
+    if (format.indexOf('Failed CompositeComponent proptype check. ') === 0) {
+      return; // Ignore CompositeComponent proptype check.
+    }
+
```

Comment:
This seems to be causing prop validation on my components to never generate warnings. I've spent a good while tracing through the validation code to end up here. If anyone reads this comment, I'm curious what the intent was here.